controllers/component_build_controller.go

```diff
@@ -342,6 +405,11 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			}
 		} else {
 			simpleBuildStatus.BuildStartTime = time.Now().Format(time.RFC1123)
+			simpleBuildPipelineCreationTimeMetric.Observe(time.Since(componentTimesForMetrics[componentIdForMetrics].startTimestamp).Seconds())
+			// initial build upon component creation
+			if initialBuildStatus.PaC == nil && initialBuildStatus.Simple == nil {
+				initialComponentCreationBuildOrPaCProvisionTimeMetric.Observe(time.Since(componentTimesForMetrics[componentIdForMetrics].startTimestamp).Seconds())
+			}
```

Comment:
What is the difference between these two metrics, other than one of them mixing both simple builds and PaC provisions? Specifically, what is the purpose of the check `if initialBuildStatus.PaC == nil && initialBuildStatus.Simple == nil`?

Reply:
Ah, so `initialBuildStatus.PaC == nil && initialBuildStatus.Simple == nil` means that the component hasn't had any initial builds or PaC provisions yet?

That makes sense. Could you add a comment to explain that?

Reply:
updated